ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/CoreLocalizationConstant.java

```diff
@@ -532,8 +532,14 @@
     @Key("projectProblem.title")
     String projectProblemTitle();
 
-    @Key("projectProblem.message")
-    String projectProblemMessage();
+    @Key("projectProblem.blank.message")
+    String projectProblemBlankMessage();
+
+    @Key("projectProblem.estimateTypes.message")
+    String projectProblemEstimateTypesMessage(String types, String keepType);
+
+    @Key("projectProblem.keepBut.title")
```

Comment:
The same https://github.com/codenvy/che-core/pull/257#discussion-diff-39620019R36